src/Resources/views/storefront/page/checkout/confirm/confirm-payment.html.twig

```diff
@@ -0,0 +1,16 @@
+{% sw_extends '@Storefront/storefront/page/checkout/confirm/confirm-payment.html.twig' %}
+
+{% block page_checkout_confirm_payment_inner %}
+    {{ parent() }}
+    {% set adyenFrontendData = page.extensions[constant('Adyen\\Shopware\\Subscriber\\CheckoutSubscriber::ADYEN_DATA_EXTENSION_ID')] %}
+    {% if adyenFrontendData and adyenFrontendData.paymentUrl %}
+        <div id="adyen-checkout-options"
+             data-language-id="{{ adyenFrontendData.languageId }}"
+             data-payment-url="{{ adyenFrontendData.paymentUrl }}"
+             data-checkout-order-url="{{ adyenFrontendData.checkoutOrderUrl }}">
+        </div>
+    {% endif %}
+    <script>
+        var adyenCheckoutOptions = document.querySelector('#adyen-checkout-options').dataset;
+    </script>
```

Comment:
Without the div, the script won't work so maybe it's better to render the script only along with the div
```suggestion
        <script>
            var adyenCheckoutOptions = document.querySelector('#adyen-checkout-options').dataset;
        </script>
    {% endif %}
```